src/include/cell_inline.h

```diff
@@ -1429,34 +1429,31 @@ __wt_cell_unpack_kv(WT_SESSION_IMPL *session, const WT_PAGE_HEADER *dsk, WT_CELL
 }
 
 /*
- * __wt_cell_unpack_delta_int --
+ * __wt_cell_unpack_delta_int_key --
  *     Unpack an internal delta cell into a structure.
  */
 static WT_INLINE void
-__wt_cell_unpack_delta_int(WT_SESSION_IMPL *session, const WT_PAGE_HEADER *page_dsk,
-  const WT_PAGE_HEADER *dsk, WT_DELTA_CELL_INT *cell, WT_CELL_UNPACK_DELTA_INT *unpack_delta)
+__wt_cell_unpack_delta_int_key(WT_SESSION_IMPL *session, const WT_PAGE_HEADER *page_dsk,
```

Comment:
This function is just calling __wt_cell_unpack_kv. Can we remove this function then?

Reply:
Fixed in 4841299c299ec85566e853eb4f7db12cb85e6408